core/src/main/java/com/google/zxing/pdf417/detector/Detector.java

```diff
@@ -77,20 +78,31 @@ public static PDF417DetectorResult detect(BinaryBitmap image, Map<DecodeHintType
     // different binarizers
     //boolean tryHarder = hints != null && hints.containsKey(DecodeHintType.TRY_HARDER);
 
-    BitMatrix bitMatrix = image.getBlackMatrix();
-
-    List<ResultPoint[]> barcodeCoordinates = detect(multiple, bitMatrix);
-    // Try 180, 270, 90 degree rotations, in that order
-    for (int rotate = 0; barcodeCoordinates.isEmpty() && rotate < 3; rotate++) {
-      bitMatrix = bitMatrix.clone();
-      if (rotate != 1) {
-        bitMatrix.rotate180();
-      } else {
-        bitMatrix.rotate90();
+    BitMatrix originalMatrix = image.getBlackMatrix();
+    for(int rotation : ROTATIONS) {
```

Comment:
Looks good. The style checker is complaining about the lack of space after `for`: `src/main/java/com/google/zxing/pdf417/detector/Detector.java:[82,5] (whitespace) WhitespaceAround: 'for' is not followed by whitespace.`

Reply:
Done, thanks!